src/lib.rs

```diff
@@ -90,6 +92,7 @@ mod tests {
     }
 
     #[test]
+    #[should_panic]
```

Comment:
This change is correct.

The reason why it was the way it was is because I have an example DMARC XML locally that the test runs against. It had too much identifying data in it to include.

I am going to try to get a DMARC file from the internet to use as, ideally, the test should see if it actually parses without panicking.